File: Day356_2025/Q356_GFG.py
class Solution:
    def rowWithMax1s(self, arr):
        n = len(arr)
        m = len(arr[0])

        max_row = -1
        max_ones = -1

        for i in range(n):
            count = 0
            for j in range(m):
                if arr[i][j] == 1:
                    count = m - j
                    break

            if count > max_ones:
                max_ones = count
                max_row = i

        return max_row

File: Day356_2025/Q356_GFG.java
class Q356_GFG {
    public int rowWithMax1s(int arr[][]) {
        int n = arr.length;
        int m = arr[0].length;

        int maxRow = -1;
        int maxOnes = -1;

        for (int i = 0; i < n; i++) {
            int count = 0;
            for (int j = 0; j < m; j++) {
                if (arr[i][j] == 1) {
                    count = m - j;
                    break;
                }
            }

            if (count > maxOnes) {
                maxOnes = count;
                maxRow = i;
            }
        }

        return maxRow;
    }
}
